overrides/default.nix

```diff
@@ -1705,6 +1717,12 @@ lib.composeManyExtensions [
         }
       );
 
+      numba = prev.numba.overridePythonAttrs (
+        old: {
+          propagatedBuildInputs = old.propagatedBuildInputs or [ ] ++ [ pkgs.tbb_2021_11 ];
```

Comment:
Is the TBB dependency necessary?

Reply:
Yes

Reply:
I don't think TBB is required: https://numba.readthedocs.io/en/stable/user/threading-layer.html#which-threading-layers-are-available

There are multiple ways to back the threading layer.

If you want to TBB with numba with poetry2nix, it's up to the end user to add tbb as a dependency. We shouldn't opt people in to optional dependencies like this. Please revert the change.

Reply:
It seems like the requirement for tbb is coming from vllm, so numba should be made to ignore missing autopatchelf deps and then you'll have to make sure in your application that uses vllm that you depend on tbb in some way.

Reply:
Fixed.